JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py

```diff
@@ -586,14 +586,25 @@ def payments_show(self, bot, reservation_create_resp, currency):
             wallet_names.append(w)
         wallet_names.append("3bot app")
 
+        explorer = j.clients.explorer.default
+        payment_details = ""
```

Comment:
Maybe the payment details message generation can be added in a separate function since it is used multiple times?

Reply:
Yes thnx